examples.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: default
+title: Example Applications
+has_children: true
+nav_order: 3
+---
+
+# Example Projects
+
+## Tupelo Explorer and Wallet
+
+Tupelo Explorer is a WASM, react-based app built on the Tupelo WASM SDK.  
+The explorer enables users to browse ChainTrees, the data-structure underlying
+Tupelo.  Tupelo Explorer can also be used as a wallet for Chaintrees (NFTs) you
```

Comment:
not really... the wallet as part of the explorer is the old one that only really does tokens.

Reply:
I should have just done it then pushed it but... this change is in the next iteration branch improving-examples .